public/script.js

```diff
@@ -22,3 +22,10 @@ function Menu(e){
       })
       .catch(error => console.error('Error!', error.message))
   })
+
+  let loader=document.getElementById("preload")
+  let loader_bg=document.getElementById("load-bg")
```

Comment:
Use camelCase for variable names.

Reading: https://www.w3schools.com/js/js_conventions.asp or any other JavaScript naming conventions article.